bundles/org.openhab.binding.epsonprojector/src/main/java/org/openhab/binding/epsonprojector/internal/EpsonProjectorException.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.epsonprojector.internal;
+
+/**
+ * Exception for Epson projector errors.
+ *
+ * @author Pauli Anttila - Initial contribution
+ */
+public class EpsonProjectorException extends Exception {
+
+    private static final long serialVersionUID = -8048415193494625295L;
+
+    public EpsonProjectorException() {
```

Comment:
Do we use all these constructors? Could you maybe trim them down a little? Also, the explicit call to super() is not needed.